docs/configuration.md

```diff
@@ -32,6 +32,7 @@ Caluma expects a bearer token to be passed on as [Authorization Request Header F
 * `OIDC_GROUPS_CLAIM`: Name of claim to be used to represent groups (default: caluma_groups)
 * `OIDC_USERNAME_CLAIM`: Name of claim to be used to represent the username (default: sub)
 * `OIDC_BEARER_TOKEN_REVALIDATION_TIME`: Time in seconds before bearer token validity is verified again. For best security token is validated on each request per default. It might be helpful though in case of slow Open ID Connect provider to cache it. It uses [cache](#cache) mechanism for memorizing userinfo result. Number has to be lower than access token expiration time. (default: 0)
+* `CALUMA_OIDC_USER_FACTORY`: User object factory. Use it to provide your own OIDC user object
```

Comment:
```suggestion
* `CALUMA_OIDC_USER_FACTORY`: User object factory. Use it to provide your own OIDC user object (default: "caluma.caluma_user.models.OIDCUser")
```
That way it's the expected format is a little more obvious.

Reply:
Also, should we call it `CALUMA_OIDC_USER_FACTORY_CLASS` to be consistent with the other extension points?

Reply:
> Also, should we call it `CALUMA_OIDC_USER_FACTORY_CLASS` to be consistent with the other extension points?

No, as the factory is just that - a factory. Could just as well be a standalone function that returns some object. The other ones are always classes of a specific type

Reply:
:+1: makes sense, thanks!

Reply:
I'll leave this "unresolved" because it was actually 2 comments in one :see_no_evil: